src/shared/extensionUtilities.ts

```diff
@@ -252,10 +312,11 @@ export function getToolkitEnvironmentDetails(): string {
     const vsCodeVersion = vscode.version
     const envDetails = localize(
         'AWS.message.toolkitInfo',
-        'OS:  {0} {1} {2}\nVisual Studio Code Version:  {3}\nAWS Toolkit Version:  {4}\n',
+        'OS:  {0} {1} {2}\n{3} Extension Host Version:  {4}\nAWS Toolkit Version:  {5}\n',
```

Comment:
Might be worth changing this to an `ExtensionProperty`

Reply:
?

Reply:
NVM, we are using it (I glanced and saw it as `isCloud9`ing the whole string, not just the long name.